### PR TITLE
Add dot files

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+language: python
+python:
+  - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+env:
+  - VERSION="8.0" ODOO_REPO="odoo/odoo"
```
Comment (pedrobaeza) [inline .travis.yml]: Please put version 7.0

**Comment by pedrobaeza** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Comment by tafaRU** [comment]
:+1:

**Pull request merged**